locales/en.json

```diff
@@ -28,6 +28,11 @@
       "spinnerAccessibilityLabel": "Loading"
     },
 
+    "ColorPicker": {
+      "textPickerAccessibilityLabel": "HEX color",
+      "alphaFieldAccessibilityLabel": "Alpha"
```

Comment:
@dpersing Any suggestions on what a11y labels should we use here?

The first is used for the hex/colour name/rgb input field and the second is used for the alpha percentage field.

Reply:
I added these as placeholders for now.